website/docs/eos-learn/eosio-protocol.md

```diff
@@ -31,22 +31,4 @@ System contracts allow EOSIO networks to :
 * Easily configure the network governance through smart contracts.
 * To have transparent rules for the operation of the system.
 * Synchronize the changes in system contracts in a instantaneous way with the whole network, which reduces the administration costs and delays of any governance updates.
-* Maintain total compatibility with other EOSIO blockchains.
-
-### EOS explained by Dan Larimer   
-
-<figure class="video_container">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/xkXuFeN-KMw" frameborder="0" allowfullscreen="true"> </iframe>
-</figure>
-
-### Introduction to EOS
-
-<figure class="video_container">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/IjrDj6KQGe8" frameborder="0" allowfullscreen="true"> </iframe>
-</figure>
-
-### Internal economy
-
-<figure class="video_container">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/2EgBTvUeH30" frameborder="0" allowfullscreen="true"> </iframe>
-</figure>
+* Maintain total compatibility with other EOSIO blockchains.
```

Comment:
@AngeloCG97  this line seems out of place

Reply:
![image](https://user-images.githubusercontent.com/51149817/87455274-6a8a5780-c5c2-11ea-91a9-5db74aa26f4e.png)

esa linea es parte del contenido original del sitio. No la agregue nueva ya estaba.